pittapi/sports.py

```diff
@@ -56,25 +48,44 @@ def get_mens_basketball_record() -> str:
     return record_summary
 
 
-def get_next_mens_basketball_game() -> NextBasketballGame:
+def get_next_mens_basketball_game() -> dict:
     """returns a dict containing details of the next scheduled men's basketball game."""
     basketball_response = requests.get(BASKETBALL_URL)
     basketball_data = basketball_response.json()
-
-    next_game = NextBasketballGame(
-        name=basketball_data["team"]["nextEvent"][0]["name"],
-        short_name=basketball_data["team"]["nextEvent"][0]["shortName"],
-        season_name=basketball_data["team"]["nextEvent"][0]["seasonType"]["name"],
-        week=basketball_data["team"]["nextEvent"][0]["week"]["text"],
-        court=basketball_data["team"]["nextEvent"][0]["competitions"][0]["venue"][
-            "fullName"
-        ],
-    )
-
-    return next_game
-
-
-def get_mens_basketball_standings():
+    next_game = None
+    try:
+        next_game = basketball_data["team"]["nextEvent"][0]
+        opponent = None
+        homeaway = None
+        status = None
+        if next_game["competitions"][0]["status"]["type"]["name"] == "STATUS_FINAL":
+            status = "GAME_COMPLETE"
+        elif next_game["competitions"][0]["status"]["type"]["name"] == "STATUS_IN_PROGRESS":
+            status = "IN_PROGRESS"
+        if next_game["competitions"][0]["competitors"][0]["id"] == 221:
+            opponent = next_game["competitions"][0]["competitors"][0]
+            homeaway = next_game["competitions"][0]["competitors"][0]["homeAway"]
+        else:
+            opponent = next_game["competitions"][0]["competitors"][1]
+            homeaway = next_game["competitions"][0]["competitors"][1]["homeAway"]
+        return {
+            "Timestamp" : next_game["date"],
```

Comment:
underscore separated keys

Reply:
Resolved in [afdce7b](https://github.com/pittcsc/PittAPI/pull/137/commits/afdce7b22c50d0c6f18af707f379a39adcacdd4a)